### ISSUE TITLE
Invalid content generated with the forEach snippet

### ISSUE THREAD

**Issue by nadeeshaan** [ballerina-platform/ballerina-lang#30438]
**Description:**
Consider the following source snippet

```
public function main() {
    Student s1 = {firstName: "Mary", lastName: "Campbel", intakeYear: 2010, gpa: 2.9};

    Student s2 = {
        firstName: "Joseph",
        lastName: "Sterling",
        intakeYear: 2010,
        gpa: 2.010
    };

    Student s3 = {
        firstName: "Christopher",
        lastName: "Nolan",
        intakeYear: 2012,
        gpa: 2.6
    };

    Student[] sArray = [s1, s2, s3];

    Report[] listResult = from var student in sArray
            where student.gpa >= 2.1
            let string degreeName = "Bachelor of Medicine",
            int graduationYear = calGraduationYear(student.intakeYear)
            order by student.firstName ascending
            limit 2
            select {
                name: student.firstName + " " + student.lastName,
                degree: degreeName,
                graduationYear: graduationYear
            };

    listResult.fo<cursor>
}
```

This source is within a multi module project and the generated snippet is invalid since it append the module name even if the record type is defined within the same source.

**Invalid Content**
```
foreach nadeeshaan/beta1_package:0.1.0:Report item in listResult {
      
}
```

**Comment by nadeeshaan** [comment]
@malinthar

**Comment by nadeeshaan** [comment]
Fix merged with #30442
